lesson_16/objects/objects_app/src/main/java/com/codedifferently/lesson16/movie/releaseYear.java

```diff
@@ -0,0 +1,8 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+
+package com.codedifferently.lesson16.movie;
+
+class releaseYear {}
```

Comment:
Class name must be `PascalCase`, not `camelCase`. Also, this doesn't seem to have a purpose.

Reply:
Yeah i'm not sure how that got there. I'll take it out